vehicles/src/common/enum/orderBy.enum.ts

```diff
@@ -27,4 +28,5 @@ export enum ApplicationOrderBy {
   UNIT_NUMBER = 'unitNumber',
   PLATE = 'plate',
   APPLICANT = 'applicant',
+  UPDATED_DATE_TIME = 'updatedDateTime',
```

Comment:
Already taken up as part of https://github.com/bcgov/onroutebc/pull/1244. Waiting for @zgong-gov FE changes for the PR to be merged. Kindly coordinate with @zgong-gov